OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -393,7 +393,7 @@ void LoadActors(IniFile file, string section)
 						players.Add(parts[0]);
 
 					var loc = Exts.ParseIntegerInvariant(parts[3]);
-					var health = float.Parse(parts[2], NumberFormatInfo.InvariantInfo) / 256;
+					var health = int.Parse(parts[2], NumberFormatInfo.InvariantInfo) * 100 / 256;
```

Comment:
They are stored as integer with 256 being full health. See http://nyerguds.arsaneus-design.com/cncstuff/mappics/campaign-gdi/maps/ for examples.